server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/MirrorSchedulingService.java

```diff
@@ -62,6 +66,19 @@ public final class MirrorSchedulingService implements MirroringService {
 
     private static final Logger logger = LoggerFactory.getLogger(MirrorSchedulingService.class);
 
+    private static final MirrorListener mirrorListener;
```

Comment:
Note) I understood that `MirrorListener` isn't invoked for manually triggered mirrors from `MirrorServiceV1`.

While this may make sense in terms of our purpose (monitoring), it may be confusing since a `Mirror` has been done yet `MirrorListener` isn't invoked.

Reply:
Good point. Let me `scheduled` flag to methods of `MirrorListener`.